### ISSUE TITLE
Can select disabled Combo Box from PDOM

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#74]
I looks like `ComboBox.enabledProperty` is only changing pickability, which works for mouse, but doesn't prevent a11y listeners from firing on the element. @jessegreenberg and I discussed this on slack and decided to pair on this tomorrow. 

It may be slightly related to https://github.com/phetsims/sun/issues/462

**Comment by zepumph** [comment]
This has been solved by https://github.com/phetsims/scenery/issues/1056. Note also that now the button has `aria-disabled` added to it when disabled.
